parser/src/test/scala/hmda/parser/fi/lar/LarGenerators.scala

```diff
@@ -91,8 +91,7 @@ trait LarGenerators extends FIGenerators {
       "30", "31", "32", "33", "34", "35", "36", "37", "38", "39",
       "40", "41", "42", "44", "45", "46", "47", "48", "49",
       "50", "51", "53", "54", "55", "56",
-      "60", "66", "69",
-      "72", "74", "78"
```

Comment:
I assume you removed 69, 74, and 78 (Northern Mariana Islands, U.S. Minor Outlying Islands, and U.S. Virgin Islands) because they are not contained in the CBSA mapping. So now this list deviates from the list linked in the comment above ([census.gov](http://www2.census.gov/geo/docs/reference/state.txt))

Could you update the comment to say what your source is? I'm also interested in why the list deviates from the [list of FIPS state codes](https://en.wikipedia.org/wiki/Federal_Information_Processing_Standard_state_code), but that may be a question for a business analyst.